src/components/Hero/Hero.tsx

```diff
@@ -22,18 +37,28 @@ export function Hero() {
         </div>
       </div>
       <div className={styles.imageWrapper}>
-        <Image
-          width={1440}
-          src={HeroDesktop}
-          height={700}
-          alt="Sam Pausha riding his bike in the Madison Ironman"
-          className={styles.heroImage}
-        />
+        {mobile ? (
```

Comment:
you'll want to use [this](https://nextjs.org/docs/pages/api-reference/components/image#sizes) instead